### PR TITLE
Merging changes from express-website to adobecom/express

### PR DESCRIPTION
**Pull request by apganapa**
Merging changes from express-website to adobecom/express

Fix 132431

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/
- After: https://mwpw-132431-from-main--express--adobecom.hlx.page/express/

### CODE REVIEW

File: express/blocks/how-to-steps-carousel/how-to-steps-carousel.css
[the remark this answers is not in the document]
Comment (vhargrave) [reply express/blocks/how-to-steps-carousel/how-to-steps-carousel.css:1]: @toka1300 that's fine - that basically means no changes. I checked 👍

[the remark this answers is not in the document]
Comment (toka1300) [reply express/blocks/how-to-steps-carousel/how-to-steps-carousel.css:1]: Odd that it shows up here then, but ok!

**Review by toka1300 — APPROVED** [review]
Looks good on my end

**Review by JingleH — APPROVED** [review]
no files i worked on are getting overwritten so it's good on my part

**Review by qiyundai — APPROVED** [review]
looks good on my end

**Pull request merged**